include/swift/AST/Ownership.h

```diff
@@ -31,21 +33,87 @@ namespace swift {
 enum class ReferenceOwnership : uint8_t {
   /// \brief a strong reference (the default semantics)
   Strong,
+  Default = Strong,
```

Comment:
Default for what?  Everywhere?

Reply:
I've removed this in a forthcoming pull request update. As an aside, I'm surprised this was commented on. The comment above it literally says "the default semantics". Also "default to strong" is fairly entrenched in the language.